src/fragments/lib-v1/project-setup/android/create-application/20_gradle.mdx

```diff
@@ -36,8 +36,8 @@ Run **Gradle Sync**
 
 Android Studio requires you to sync your project with your new configuration. To do this, click **Sync Now** in the notification bar above the file editor.
 
-![](/images/lib/getting-started/android/set-up-android-studio-sync-gradle.png)
+![Shows how to sync project with gradle](/images/lib/getting-started/android/set-up-android-studio-sync-gradle.png)
```

Comment:
The phrasing of this sentence using the word "how" can make it unclear since users won't actually know what's being shown in the picture. We could make it decorative if you want (since the text above it explains what to do) or we could try and describe it with some more detail. Something like, "Screenshot with an arrow pointing to the Sync Now button in the file's notification bar"

Reply:
Done